Ktisis/Interface/Components/BoneTree.cs

```diff
@@ -7,14 +7,31 @@
 using Ktisis.Structs;
 using Ktisis.Structs.Bones;
 using Ktisis.Structs.Actor;
+using Ktisis.Util;
+using Dalamud.Interface;
+using System;
 
 namespace Ktisis.Interface.Components {
 	public class BoneTree {
+		private enum HighlightCriteria {
```

Comment:
I'm not sure `Criteria` is the right term to use here, though I don't know what a good alternative would be. Possibly `HighlightReason` or something to that effect?